src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -236,6 +239,49 @@ public ObservableCollection<string> LanguagesList
             }
         }
 
+        /// <summary>
+        /// PackagePathsForInstall contains the list of all package paths where
+        /// packages can be installed.
+        /// </summary>
+        public ObservableCollection<string> PackagePathsForInstall
+        {
+            get
+            {
+                if (packagePathsForInstall == null || !packagePathsForInstall.Any())
+                {
+                    packagePathsForInstall = new ObservableCollection<string>();
+                    packagePathsForInstall.AddRange(preferenceSettings.CustomPackageFolders.Where(
+                        x => x != DynamoModel.StandardLibraryToken));
```

Comment:
WIP: I need to filter out paths that include filenames like `...\myAssembly.dll`

Reply:
nice catch.